src/client/features/enrichment/token-input.js

```diff
@@ -24,12 +24,6 @@ class TokenInput extends React.Component {
     let { inputBoxContents } = this.state;
     let { controller } = this.props;
 
-    if(!inputBoxContents) return;
-    else if(!inputBoxContents.replace(/\s/g, "")) {
```

Comment:
Sorry I meant that you only had to remove the first part of the else if.

You can return early if the replace check passes.

```js
if(!inputBoxContents.replace(/\s/g, "")) {
   this.setState({ inputBoxContents: "" });	
   return;
}
```